src/Compilers/CSharp/Test/Syntax/Parsing/LambdaParameterParsingTests.cs

```diff
@@ -290,44 +274,47 @@ class C {
                                     }
                                     N(SyntaxKind.VariableDeclarator);
                                     {
-                                        N(SyntaxKind.IdentifierToken);
+                                        N(SyntaxKind.IdentifierToken, "f");
                                         N(SyntaxKind.EqualsValueClause);
                                         {
                                             N(SyntaxKind.EqualsToken);
-                                            N(SyntaxKind.ParenthesizedLambdaExpression);
+                                            N(SyntaxKind.ParenthesizedExpression);
                                             {
-                                                N(SyntaxKind.ParameterList);
-                                                {
-                                                    N(SyntaxKind.OpenParenToken);
-                                                    N(SyntaxKind.Parameter);
-                                                    {
-                                                        N(SyntaxKind.OutKeyword);
-                                                        N(SyntaxKind.IdentifierName); // parameter type
-                                                        {
-                                                            N(SyntaxKind.IdentifierToken);
-                                                        }
-                                                        N(SyntaxKind.IdentifierToken); // parameter name
-                                                    }
-                                                    M(SyntaxKind.CloseParenToken);
-                                                }
-                                                M(SyntaxKind.EqualsGreaterThanToken);
-                                                M(SyntaxKind.IdentifierName); // lambda body
+                                                N(SyntaxKind.OpenParenToken);
+                                                M(SyntaxKind.IdentifierName);
                                                 {
                                                     M(SyntaxKind.IdentifierToken);
                                                 }
+                                                M(SyntaxKind.CloseParenToken);
                                             }
                                         }
                                     }
                                 }
                                 M(SyntaxKind.SemicolonToken);
                             }
+                            N(SyntaxKind.LocalDeclarationStatement);
```

Comment:
I anticipate this is going to cause all sorts of hell for completion. Not sure what to do about it, but it won't be fun.